### ISSUE TITLE
Support Gmailify?

### ISSUE THREAD

**Issue by shaunobi** [ScottPeterJohnson/purelymail-issues#256]
Not sure what's involved, but would be useful to connect Gmail to my Purelymail custom domain via IMAP, and whatever Gmailify requires.

**Comment by femme-theory23** [comment]
Yeah, I agree. The Gmail interface is quite nice to use sometimes, insecure but thats a different matter.

**Comment by ScottPeterJohnson** [comment]
Hm, I can't seem to find any information on how to get a provider working with Gmailify. Maybe it's only for the big providers they added explicit support for?
You can kind of get the pulling email into Gmail through POP3 and the sending email on your custom domain already though, you just wouldn't have the provider sync that Gmailify seems to do.

**Comment by wilcooo** [comment]
There's two things called Gmailify:
1. https://www.gmailify.com/ that forwards mails to your Gmail inbox. You could use that instead of purelymail (or possibly forward from purely to gmailify to google but I don't know why anyone would do that).

2. Google's own more recent [Gmailify](https://support.google.com/mail/answer/6304825) that only works for Yahoo, AOL, Outlook, Hotmail, or select non-Gmail accounts. I don't think there's any chance Google will add purelymail to that list.

The only thing that purely *could* improve is steal the [approach of the first gmailify](https://www.gmailify.com/docs#t1-4), where it allows forwarding to Gmail (as it already does) and on top of that allow POP3 access *only* for the mails that were rejected during forwarding.

**Comment by Mr-McMuffin** [comment]
This would be amazing, it allows us to offset webmail with a super effective service. Less spam too!

**Comment by amerikiwi93** [comment]
I really like this idea however it looks like both options are no longer available; gmailify.com cannot be accessed and Google's Gmailify service is being sunsetted [Gmailify and POP are going away: r/GMail](https://www.reddit.com/r/GMail/comments/1rxaxgz/heads_up_gmailify_and_pop_are_going_away_and_this/).

Apparently cPanel introduced experimental support for Authenticated Received Chain (ARC) to help secure forwarded emails and improve deliverability for Gmail. Perhaps this could be an alternative solution?

**Comment by HardToFindaUserName1** [comment]
I would like to import emails into Rc & continue to fetch them as forwarding from outlook/gmail to Pm is way better than forwarding to Outlook from Gmail it is still not completely reliable. Thanks.

**Comment by femme-theory23** [comment]
Regarding ARC stuff, this is something that I am interested in looking into. However, a lot of the research that I would need to do for this would be looking through a very high page count RFC document, which I don't believe is even technically out of "beta". It does seem widely supported though, so I will try to stick with it, as long and boring as it gets 😓
